### ISSUE TITLE
Smart install considers only first source for newly referenced packages

### ISSUE THREAD

**Issue by mexx** [fsprojects/Paket#715]
Given following `paket.dependencies`
```
source http://custom/...
source https://www.nuget.org/api/v2
```

Put any NuGet package from second source into `paket.references` file.
```
Newtonsoft.Json
```

A run of `paket install` results in:
```
Paket failed with:
        Could not retrieve data from http://custom/.../Packages?$filter=Id eq 'Newtonsoft.Json'
 Message: The remote name could not be resolved: 'custom'
```

**Comment by forki** [comment]
sorry I can't reproduce. can you give more details?

**Comment by mexx** [comment]
I've created a repro: https://github.com/mexx/Paket.Repros/tree/reproduce-715

**Comment by forki** [comment]
thanks. I can now reproduce

**Comment by forki** [comment]
fixed in 0.35.12
